app/shared/translate/lang-de.ts

```diff
@@ -0,0 +1,10 @@
+﻿/**
+ * German Tanslations goes here.
+ */
+
+export const LANG_DE_NAME = 'de';
+
+export const LANG_DE_TRANS = {
+    'hello world': 'hallo welt',
+    'login title': 'Geben Sie Ihre Daten einloggen',
```

Comment:
Should be:
'Geben Sie Ihre Daten ein'

But I think we have to discuss if we will use "du" or "Sie"